daemon/src/model/cfd.rs

```diff
@@ -1352,13 +1354,61 @@ pub fn calculate_profit_at_price(
 
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 pub struct Cet {
-    pub tx: Transaction,
+    #[serde(with = "::bdk::bitcoin::util::amount::serde::as_sat")]
+    pub maker_amount: Amount,
+    #[serde(with = "::bdk::bitcoin::util::amount::serde::as_sat")]
+    pub taker_amount: Amount,
     pub adaptor_sig: EcdsaAdaptorSignature,
 
     // TODO: Range + number of digits (usize) could be represented as Digits similar to what we do
     // in the protocol lib
     pub range: RangeInclusive<u64>,
     pub n_bits: usize,
```

Comment:
It is just one number but isn't this also duplicates across all CETs for one attestation?

Reply:
Actually, I was wrong when I thumbed up before. This number represents the number of  _relevant_ bits, which changes depending on the range being represented. It's used so that we can zip with the right nonces and attestations from the oracle.

Looking at it now, it seems to me like an optimisation (although I'm not sure I'm remembering everything). We should be able to multiply nonces/attestations by zero without affecting the total sum. In any case, I wouldn't tackle this now.

Reply:
> Looking at it now, it seems to me like an optimisation (although I'm not sure I'm remembering everything). We should be able to multiply nonces/attestations by zero without affecting the total sum. In any case, I wouldn't tackle this now.

Okay, can you record this as an idea for later optimization?

Reply:
What I was trying to say is that we appear (not 100% sure) to have optimised for fewer operations (by removing multiplications by zero). But we might want to reconsider that choice in favour of storing less information. I'll create an issue.

Reply:
See https://github.com/itchysats/itchysats/issues/1301.